client/sources/ok_test/models.py

```diff
@@ -100,9 +100,11 @@ def unlock(self, interact):
                                 for case in suite.cases])
         for suite_num, suite in enumerate(self.suites):
             for case_num, case in enumerate(suite.cases):
+                case_id = '{} > Suite {} > Case {}'.format(
```

Comment:
If you really want to add this hash, add it in addition to the
human-readable identifier based on the position. Human-readable question
identifiers will make everyone's life easier in the long run.

On Mon, Jun 29, 2015 at 3:59 PM, soumyabasu <notifications@github.com>
wrote:

> In client/sources/ok_test/models.py
> <https://github.com/Cal-CS-61A-Staff/ok-client/pull/97#discussion_r33524132>
> :
>
> > @@ -100,9 +100,11 @@ def unlock(self, interact):
> >                                  for case in suite.cases])
> >          for suite_num, suite in enumerate(self.suites):
> >              for case_num, case in enumerate(suite.cases):
> > +                case_id = '{} > Suite {} > Case {}'.format(
>
> Oh wait, sorry. That's not quite right. I misread the first prompt. The
> prompt for the first thing would be:
>
> ">>> x=5\n>>> square(x)\nhash(25)\n>>>square(-x)\nhash(25)"
>
> The prompt for the second thing is correct. And yeah, this can be
> implemented easily since we have access to the full text on each unlocking
> answer.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Cal-CS-61A-Staff/ok-client/pull/97/files#r33524132>.
>